[2025-03-21T21:32:15.394+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T21:32:15.408+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coldpath_ingestion.ingest_cold_twitter_data manual__2025-03-21T16:31:22.588046+00:00 [queued]>
[2025-03-21T21:32:15.414+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coldpath_ingestion.ingest_cold_twitter_data manual__2025-03-21T16:31:22.588046+00:00 [queued]>
[2025-03-21T21:32:15.414+0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-21T21:32:15.434+0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): ingest_cold_twitter_data> on 2025-03-21 16:31:22.588046+00:00
[2025-03-21T21:32:15.441+0500] {standard_task_runner.py:72} INFO - Started process 114324 to run task
[2025-03-21T21:32:15.445+0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'coldpath_ingestion', 'ingest_cold_twitter_data', 'manual__2025-03-21T16:31:22.588046+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/coldpath.py', '--cfg-path', '/tmp/tmpm9c9_z9m']
[2025-03-21T21:32:15.446+0500] {standard_task_runner.py:105} INFO - Job 34: Subtask ingest_cold_twitter_data
[2025-03-21T21:32:15.482+0500] {task_command.py:467} INFO - Running <TaskInstance: coldpath_ingestion.ingest_cold_twitter_data manual__2025-03-21T16:31:22.588046+00:00 [running]> on host ubuntu
[2025-03-21T21:32:15.542+0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='coldpath_ingestion' AIRFLOW_CTX_TASK_ID='ingest_cold_twitter_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T16:31:22.588046+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T16:31:22.588046+00:00'
[2025-03-21T21:32:15.543+0500] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-21T21:32:15.543+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-21T21:32:15.543+0500] {logging_mixin.py:190} INFO - Current task name:ingest_cold_twitter_data state:running start_date:2025-03-21 16:32:15.408492+00:00
[2025-03-21T21:32:15.543+0500] {logging_mixin.py:190} INFO - Dag name:coldpath_ingestion and current dag run status:running
[2025-03-21T21:32:15.543+0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T21:32:50.798+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:32:50.798+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:32:55.846+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:32:55.846+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:00.902+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:00.902+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:05.916+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:05.917+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:10.972+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:10.973+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:16.020+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:16.021+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:21.067+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:21.068+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:26.110+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:26.110+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:31.173+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:31.174+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:36.236+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:36.237+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:41.283+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:41.283+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:46.332+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:46.333+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:51.384+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:51.384+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:33:56.428+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:33:56.428+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:01.479+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:01.480+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:06.524+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:06.524+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:11.571+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:11.572+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:16.619+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:16.619+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:21.678+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:21.679+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:26.725+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:26.726+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:31.786+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:31.786+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:36.804+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:36.804+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:41.857+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:41.858+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:46.878+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:46.878+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:51.898+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:51.899+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:34:56.918+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:34:56.918+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:01.984+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:01.985+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:07.033+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:07.034+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:12.099+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:12.100+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:17.144+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:17.144+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:22.191+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:22.192+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:27.235+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:27.236+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:32.282+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:32.282+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:37.325+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:37.326+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:42.379+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:42.380+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:47.433+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:47.433+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:52.486+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:52.486+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:35:57.523+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:35:57.524+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:02.559+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:02.560+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:07.605+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:07.606+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:12.641+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:12.642+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:17.706+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:17.707+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:22.764+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:22.764+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:27.784+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:27.784+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:32.797+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:32.798+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:37.862+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:37.863+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:42.933+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:42.935+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:47.960+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:47.961+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:53.006+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:53.007+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:36:58.035+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:36:58.036+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:03.066+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:03.067+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:08.125+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:08.125+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:13.151+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:13.152+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:18.175+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:18.175+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:23.237+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:23.237+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:28.267+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:28.267+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:33.306+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:33.307+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:38.356+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:38.357+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:43.376+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:43.377+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:48.398+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:48.398+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:53.470+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:53.471+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:37:58.519+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:37:58.520+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:03.570+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:03.570+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:08.637+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:08.638+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:13.703+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:13.704+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:18.759+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:18.759+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:23.832+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:23.832+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:28.884+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:28.884+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:33.929+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:33.929+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:38.976+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:38.976+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:44.029+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:44.030+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:49.109+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:49.111+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:54.159+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:54.160+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:38:59.198+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:38:59.199+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:04.270+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:04.271+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:09.335+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:09.337+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:14.354+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:14.354+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:19.375+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:19.376+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:24.396+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:24.397+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:29.437+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:29.438+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:34.456+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:34.457+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:39.534+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:39.535+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:44.598+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:44.600+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:49.658+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:49.660+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:54.677+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:54.678+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:39:59.705+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:39:59.705+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:04.758+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:04.759+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:09.825+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:09.826+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:14.866+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:14.868+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:19.905+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:19.906+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:24.966+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:24.967+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:30.022+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:30.023+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:35.048+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:35.049+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:40.070+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:40.070+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:45.107+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:45.108+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:50.172+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:50.173+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:40:55.204+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:40:55.205+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:00.277+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:00.278+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:05.300+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:05.300+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:10.330+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:10.331+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:15.362+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:15.363+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:20.395+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:20.395+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:25.423+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:25.424+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:30.474+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:30.474+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:35.536+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:35.537+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:40.570+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:40.571+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:45.638+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:45.638+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:50.695+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:50.696+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:41:55.726+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:41:55.727+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:42:00.746+0500] {local_task_job_runner.py:318} WARNING - Recorded pid 114496 does not match the current pid 114324
[2025-03-21T21:42:00.746+0500] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2025-03-21T21:42:02.770+0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T21:42:02.775+0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T21:42:02.776+0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=coldpath_ingestion, task_id=ingest_cold_twitter_data, run_id=manual__2025-03-21T16:31:22.588046+00:00, execution_date=20250321T163122, start_date=20250321T163245, end_date=20250321T164202
[2025-03-21T21:42:02.798+0500] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-21T21:42:02.798+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-21T21:42:02.799+0500] {logging_mixin.py:190} INFO - Dag name:coldpath_ingestion queued_at:2025-03-21 16:31:22.602316+00:00
[2025-03-21T21:42:02.799+0500] {logging_mixin.py:190} INFO - Task hostname:ubuntu operator:PythonOperator
[2025-03-21T21:42:02.813+0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T21:42:02.821+0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T21:42:02.822+0500] {local_task_job_runner.py:245} INFO - ::endgroup::
